rules/linux/execution_vi_binary.toml

```diff
@@ -25,8 +26,8 @@ type = "eql"
 
 query = '''
 sequence by host.id,process.pid with maxspan=1m
-[process where process.name == "vi" and process.args : "-c" and process.args : (":!/bin/bash", ":!/bin/sh", ":!bash", ":!sh")]
-[process where process.parent.name == "vi" and process.name : ("bash", "sh")]
+[process where process.name : ("vi", "vim") and process.args : "-c" and process.args : (":!/bin/bash", ":!/bin/sh", ":!bash", ":!sh")]
```

Comment:
why the change from `:` to `==`? `==` is more accurate for this case (on Linux)

Reply:
++